src/Compilers/CSharp/Portable/Lowering/MethodToClassRewriter.cs

```diff
@@ -577,7 +577,12 @@ public override BoundNode VisitLoweredConditionalAccess(BoundLoweredConditionalA
                 return null;
             }
 
-            if (method.ContainingType.IsAnonymousType)
+            if (method.ContainingType is null)
```

Comment:
>method.ContainingType is null [](http://example.com/codeflow?start=16&length=29)

Is this case for the "Throw" methods? I'm curious why this wasn't needed for the "ComputeStringHash" method.

Reply:
It's because in that case, we never include the method symbol in the bound tree, and simply grab the CCI adapter for the symbol and use it directly in the emit layer￼.